core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -131,8 +130,8 @@ public void play() throws IllegalStateException {
     }
 
     @Override
-    public void play(VideoPosition position) throws IllegalStateException {
-        seekTo(position);
+    public void play(long positionInMillis) throws IllegalStateException {
```

Comment:
Given we're changing the APIs, can we rename this method to `playAt(long)` or `playFrom(long)`?

Reply:
Yes of course 😄  happy to do any interface changes that anyone suggests. This will be a major release given the breaking changes.